src/avatar.scss

```diff
@@ -0,0 +1,172 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+/*!
+.fd-avatar+((--circle), (--thumbnail), (--xxs|--xs|--s|--m|--l|--xl|--xxl))
+*/
+$block: #{$fd-namespace}-avatar;
+
+.#{$block} {
+  // LOCAL VARS (set all themeable properties, always include !default)
+  $fd-const-ratio: 1rem;
+  $fd-avatar-sizes: (
+    xs: $fd-const-ratio * 2,
+    s: $fd-const-ratio * 3,
+    m: $fd-const-ratio * 4,
+    l: $fd-const-ratio * 5,
+    xl: $fd-const-ratio * 7,
+  ) !default;
+  $fd-avatar-border-radius: $fd-border-radius !default;
+  $fd-avatar-border-fade: rgba(0, 0, 0, 0.5); // Should be fade(var(--sapContent_ShadowColor), 10);
```

Comment:
Is the comment there because we are missing a theming variable?

Reply:
No, fades do not work with box-shadow.  I saw a similar replacement in dialog.scss so I just copied that

Reply:
we should open an issue to track the issue and fix it once the theme designer package introduce a variable